### PR TITLE
added circular linklist

### PR DESCRIPTION
**Pull request by jsroyal**
update with small info

### CODE REVIEW

**Comment by kdn251** [comment]
hey, thank you for the update! Super minor things, but could you insert the things in parentheses
Singly-linked list: linked list in which nodes have pointers to the next node and (the) last node point(s) to null
Doubly-linked list: linked list in which nodes have two pointers (which) point (to) *both* the previous node and the next node
Circular-linked list: linked list in which nodes have pointers to the next node and (the) last node point(s) (to the) head

**Comment by jsroyal** [comment]
sure!

File: README.md [hunk 1]
```diff
@@ -30,9 +30,9 @@ Interviews
  * A *Linked List* is a linear collection of data elements, called nodes, each
    pointing to the next node by means of a pointer. It is a data structure
    consisting of a group of nodes which together represent a sequence.
- * **Singly-linked list**: linked list in which nodes have pointers to the next node and last node point to null
- * **Doubly-linked list**: linked list in which nodes have two pointers to point *both* the previous node and the next node
- * **Circular-linked list**: linked list in which nodes have pointers to the next node and last node point again head
+ * **Singly-linked list**: linked list in which nodes have pointers to the next node and the last node point's to null
+ * **Doubly-linked list**: linked list in which nodes have two pointers which point's to *both* the previous node and the next node
```
Comment (EugenHotaj) [inline README.md]: Actually this line should read "linked list in which nodes have two pointers which point to *both* the previous and the next node

Comment (jsroyal) [inline README.md]: @EugenHotaj  Yeap! agree here need points instead of point's.Rest of about what?

**Comment by jsroyal** [comment]
@kdn251 Anyway, Is it fine now?

File: README.md [hunk 2]
```diff
@@ -31,7 +31,7 @@ Interviews
    pointing to the next node by means of a pointer. It is a data structure
    consisting of a group of nodes which together represent a sequence.
  * **Singly-linked list**: linked list in which nodes have pointers to the next node and the last node points to null
- * **Doubly-linked list**: linked list in which nodes have two pointers to points to *both* the previous node and the next node
+ * **Doubly-linked list**: linked list in which nodes have two pointers which points to *both* the previous node and the next node
```
Comment (EugenHotaj) [inline README.md]: Change "points" to "point"

Comment (kdn251) [inline README.md]: Yep like @EugenHotaj  said. After that, I can merge.

Comment (jsroyal) [inline README.md]: @EugenHotaj  
 > Change "points" to "point" (only in this sentence, the rest are ok).

Which sentence?

**Comment by kdn251** [comment]
I believe the grammar is still incorrect unfortunately, if you can touch it up, I'm happy to merge the changes!

**Comment by jsroyal** [comment]
@kdn251 Sure! please go ahead.Which accent do you have UK or US.
I have Indian accent so please go head.

**Comment by jsroyal** [comment]
Hey! I thought you merged. Still in waiting why?
I'm existing to contribute your repo but it seems wired because of languages barriers. 
It's okay.
Bust luck buddy.

**Comment by EugenHotaj** [comment]
Hey @jsroyal, yea language barriers really suck. I can't even imagine trying to push something in another language. Maybe you can push some code instead :).

**Comment by jsroyal** [comment]
@EugenHotaj yeah! I agree, Can reivew it or still need something improvement.

File: README.md [hunk 3]
```diff
@@ -30,8 +30,9 @@ Interviews
  * A *Linked List* is a linear collection of data elements, called nodes, each
    pointing to the next node by means of a pointer. It is a data structure
    consisting of a group of nodes which together represent a sequence.
- * **Singly-linked list**: linked list in which nodes have pointers to the next node
- * **Doubly-linked list**: linked list in which nodes have pointers to *both* the previous node and the next node
+ * **Singly-linked list**: linked list in which nodes have pointers to the next node and the last node point to null
```
Comment (EugenHotaj) [inline README.md]: "linked list in which each node points to the next node and the last node points to null"

File: README.md [hunk 4]
```diff
@@ -30,8 +30,9 @@ Interviews
...
+ * **Doubly-linked list**: linked list in which nodes have two pointers which points to *both* the previous node and the next node
```
Comment (EugenHotaj) [inline README.md]: "linked list in which each node has two pointers *p*, *n* such that *p* points to the previous node and *n* points to the next node; the last node's *n* pointer points to null"

File: README.md [hunk 5]
```diff
@@ -30,8 +30,9 @@ Interviews
...
+ * **Circular-linked list**: linked list in which nodes have pointers to the next node and the last node points again to the head
```
Comment (EugenHotaj) [inline README.md]: "linked list in which each node points to the next node and the last node points back to the first node"

**Comment by jsroyal** [comment]
Hey! @kdn251 merge it.

**Comment by kdn251** [comment]
Thanks for contributing sorry that it was so tedious. Thanks for your help @EugenHotaj

**Comment by 1348168417** [comment]
Hi what's your name!!!1

**Pull request merged**